modules/overview-of-openshift-gitops.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="overview-of-openshift-gitops_{context}"]
+= Overview of OpenShift GitOps
+
+Red Hat OpenShift GitOps is a set of practices that use Git pull requests to manage infrastructure and application configurations. The Git repository in Red Hat OpenShift GitOps is the only source of truth for system and application configuration. The repository contains the entire state of the system so that the trail of changes to the system state are visible and auditable.
+
+Use Red Hat OpenShift GitOps tooling to create repeatable and predictable processes for managing and recreating OpenShift Container Platform clusters and applications. By using Red Hat OpenShift GitOps, you resolve the issues of infrastructure and application configuration sprawl.
+
+Red Hat OpenShift GitOps defines infrastructure and application definitions as “code.” Then, Red Hat OpenShift GitOps uses this code to manage multiple workspaces and clusters and to simplify the creation of infrastructure and application configurations. By following the code’s principles, you can store the configuration of clusters and applications in Git repositories then follow the Git workflow to apply these repositories to your chosen clusters. Red Hat OpenShift GitOps ensures consistency in applications when you deploy them to different clusters in different environments, like: development, stage, and production.
```

Comment:
s/Red Hat OpenShift GitOps/ GitOps througout this paragraph.
"code". period outside the quotation mark.

Make this sentence the last one in this paragraph:
You can apply the core principles of developing and maintaining software in a Git repository to the creation and management of your cluster and application configuration files.

Remove this line from this paragraph and add it to the next:
Red Hat OpenShift GitOps ensures consistency in applications when you deploy them to different clusters in different environments, like: development, stage, and production.